src/doppelganger/CollectorsChoice.sol

```diff
@@ -169,7 +169,7 @@ contract CollectorsChoice is ERC1967Proxy {
 
         store.tokens[tokenId] = tokenUriIndex;
 
-        emit URIChanged(msg.sender, tokenId, store.uris[tokenUriIndex]);
+        emit MetadataUpdate(tokenId);
```

Comment:
is URIChanged more accurate language here?

Reply:
MetadataUpdate will trigger automatic refreshed on OpenSea and is specified by ERC-4906 so this is much better to use.